### ISSUE TITLE
App will not close with 'system' frame style on Linux Mint

### ISSUE THREAD

**Issue by WhosyVox** [kimlimjustin/xplorer#257]
### Description

When using 'system default' frame style, the close button does not close the application.
I have only tested with the latest AppImage build.

### Steps To Reproduce

1. Launch the latest AppImage build of Xplorer on Linux Mint (or I suspect Ubuntu and other distros)
2. Go to settings - Appearance
3. Switch 'Frame Style' to 'System Default'
4. Attempt to close the app using the close button on the title bar

### Expected behavior

The application should close.

### Xplorer Version

v1.0.0-beta.8

### Operating System Version

Linux Mint 20.3

**Comment by Looky1173** [comment]
This is also an issue on Windows 10.

**Comment by kimlimjustin** [comment]
Does reopening the app and close it with the 'system frame' also not working?

**Comment by Looky1173** [comment]
> Does reopening the app and close it with the 'system frame' also not working?

No; after stopping and reopening Xplorer in Task Manager with the "system tray" option, closing it again with the Windows 10 default close button still does not work.

**Comment by WhosyVox** [comment]
> Does reopening the app and close it with the 'system frame' also not working?

It's the same case here, restarting the app will not affect the functionality (or lack of) for the system close button.
